salt/fileserver/gitfs.py

```diff
@@ -50,7 +50,7 @@
 
 PER_REMOTE_OVERRIDES = ('base', 'mountpoint', 'root', 'ssl_verify',
                         'env_whitelist', 'env_blacklist', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'saltenv')
+PER_REMOTE_ONLY = ('formula', 'name', 'saltenv')
```

Comment:
I thought we discussed naming this as ``global``, not ``formula``. This feature is useful for more than just formulas, so giving it such a specific name seems wrong.